snorkel/learning/gen_learning.py

```diff
@@ -210,11 +168,42 @@ class prior (in log scale)
             L = sparse.csr_matrix(L)
         L = L[idxs, :]
 
+        # Optionally: Remap the values of L (in categorical setting) so that the
```

Comment:
@stephenbach any thoughts on what to do if there are labels outside of the specified range for a candidate?  Default is just throw an error (what I plan to do for now), but also could union with the support set / leave this as an option...